help_test.go

```diff
@@ -1096,6 +1097,7 @@ func TestHideHelpCommand_RunAsSubcommand_False(t *testing.T) {
 		t.Errorf("Run returned unexpected error: %v", err)
 	}
 }
+*/
```

Comment:
Are you planning to add (back) a compatible `RunAsSubcommand` method so that the API is retained?

Reply:
Added. Thanks for pointing this out.